src/components/Users/Users.js

```diff
@@ -107,7 +111,7 @@ Users.defaultProps = {
   onPageChanged: PropTypes.func,
   currentPage: PropTypes.number,
   totalUsersCount: PropTypes.number,
-  followingInProgress: PropTypes.bool,
+  followingInProgress: PropTypes.instanceOf(Array),
```

Comment:
Consider using "PropTypes.arrayOf()" in combination with "PropTypes.shape()";

Reply:
I would prefer to do this upon completion of priority tasks